jdaviz/tests/test_subsets.py

```diff
@@ -690,7 +690,7 @@ def test_only_overlapping_spectral_regions(specviz_helper, spectrum1d):
 def test_overlapping_in_specviz2d(specviz2d_helper, mos_spectrum2d):
     specviz2d_helper.load_data(spectrum_2d=mos_spectrum2d)
     viewer = specviz2d_helper.app.get_viewer(
-        specviz2d_helper._default_spectrum_2d_viewer_reference_name)
+        specviz2d_helper._default_spectrum_viewer_reference_name)
```

Comment:
I don't understand the intention of this test. The rest of the test looks like it is targeting the 1D spectrum viewer, with all the Angstrom values. If you indeed was meaning to test the 1D viewer, I think the entire test needs to be rewritten to work in pixels.

Reply:
Well, I think previously making a subset in the 2D viewer would also show the subset in the 1D viewer, but it seems like that's broken or intentionally decoupled now (I don't remember off the top of my head). So maybe it was testing that? Either way, we want pixels now from the 2D viewer subsets so I think this makes sense. @javerbukh wrote this test so maybe he can confirm.

Reply:
Actually this change is gone. I think Jesse removed this in his PR. Nvm.